Sources/AblyChat/Room.swift

```diff
@@ -20,23 +20,34 @@ public protocol Room: AnyObject, Sendable {
 internal actor DefaultRoom: Room {
     internal nonisolated let roomID: String
     internal nonisolated let options: RoomOptions
+    private let chatAPI: ChatAPI
+
+    private let _messages: any Messages
```

Comment:
I would tend to agree with you but we're using a pattern in which we're exposing the same property publicly without the `_` so the notation here makes sense

Reply:
Yeah I agree, it's not clear to me here why we need a separate property? I think that elsewhere when we've used an underscored property it's so that this property can have a concrete type to allow us to internally call some `internal` methods, but here `_messages` has the same type as `messages` (i.e. `any Messages`)

Reply:
fixed... previous implementation/skeleton had these computed properties throwing fatalError... didn't put 2 and 2 together and remove it if possible